projects/plugins/jetpack/modules/waf.php

```diff
@@ -0,0 +1,13 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Module Name: Firewall
+ * Module Description: Protect your site with Jetpack's Web Application Firewall
+ * Sort Order: 5
+ * First Introduced: 10.9
+ * Requires Connection: No
```

Comment:
This requires a connection since we get the rules from the API.

```suggestion
 * Requires Connection: Yes
```